File: src/main/java/com/itsci/mju/maebanjumpen/dto/ReviewDTO.java
package com.itsci.mju.maebanjumpen.dto;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor; // <--- FIX: Added for Jackson deserialization

import java.time.LocalDateTime;
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class ReviewDTO {

    private Integer reviewId;
    private String reviewMessage;
    private Double score;
    private LocalDateTime reviewDate;
    private Integer  hireId;
    private String hirerFirstName;
    private String hirerLastName;
    private String hirerPictureUrl;

}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/review/dto/ReviewDTO.kt
package com.itsci.mju.maebanjumpen.review.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.itsci.mju.maebanjumpen.hire.dto.HireDTO
import java.time.LocalDateTime

@JsonIgnoreProperties(ignoreUnknown = true)
data class ReviewDTO(
  var id: Long? = 0,
  var reviewMessage: String? = null,
  var score: Double? = null,
  var reviewDate: LocalDateTime? = null,
  var hires: HireDTO? = null
)

